internal/cli/atlas/deployments/setup.go

```diff
@@ -363,9 +362,34 @@ func (opts *SetupOpts) configureMongot(ctx context.Context, keyFileContents stri
 			mongotMetricsVolume: "/var/lib/mongot/metrics",
 		},
 		Network: opts.LocalNetworkName(),
-	})
+	}); err != nil {
+		return err
+	}
 
-	return err
+	for { // wait for mongot
+		select {
+		case <-ctx.Done():
+			return ctx.Err()
+		default:
+			_, err := opts.podmanClient.RunContainer(ctx, podman.RunContainerOpts{
```

Comment:
instead of running a new container we can also execute the same command in the running mongod container

Reply:
I've tried it but running it from golang exec package I run into https://github.com/containers/podman/issues/11197, so keeping it as is for now

Reply:
addressed in latest commit